lab_6_pipeline/pipeline.py

```diff
@@ -99,6 +176,8 @@ def analyze(self, texts: list[str]) -> list[StanzaDocument | str]:
         Returns:
             list[StanzaDocument | str]: List of documents
         """
+        docs = [self._analyzer(text)._.conll_str for text in texts]
+        return docs
```

Comment:
can we return list comprehension directly?

Reply:
done